engine/tx_sync_task.go

```diff
@@ -0,0 +1,132 @@
+package engine
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/chainstate"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/rs/zerolog"
+)
+
+// delayForBroadcastedTx indicates the time after which a broadcasted transaction should be checked
+// most probably ARC callback hasn't been received in this time so we need to check the transaction status "manually"
+func delayForBroadcastedTx() time.Time {
+	return time.Now().Add(-time.Hour)
```

Comment:
I don't thing it would make the code more readable. Moreover, the function additionally does the `time.Now().Add` stuff.
Consider this:
```golang
const delayForBroadcastedTxDuration = time.Hour
func delayForBroadcastedTx() time.Time {
	return time.Now().Add(-delayForBroadcastedTxDuration )
}
```

Is it more readable?

Reply:
Okay, you are right, my thought was to name it to have a thought when reading:
-> okay 10mins specified here is because it's averaged time for mining a new block.

But maybe it's unnecessary